frontend/src/utils/Button.svelte

```diff
@@ -0,0 +1,37 @@
+<style>
+  button {
+    display: flow-root;
```

Comment:
Why this display?

Reply:
Adding flow-root treats the button as a single element, making margin + padding + size = the size seen by parent blocks, got rid of some of the padding overflow issues i was having.

Reply:
was also my solution to margin collapse